folly/concurrency/test/AtomicSharedPtrPerformance.cpp

```diff
@@ -13,6 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+// AtomicSharedPtr-detail.h only works with libstdc++, so skip these tests for other vendors
```

Comment:
Use an 80-column ruler.

(Saves me the step of auto-formatting this after import.)